---
layout: default
title: Publications | Evolutionary genomics of pathogens
permalink: /publications/
---
<script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

# Publications

[Preprints on BioRχiv](#preprints-on-biorχiv)  
[Preprints and datasets on Zenodo](#preprints-and-datasets-on-zenodo)  
[Publications 2022 and in press](#publications-2022-and-in-press)  

(lab members in bold)

## Selected recent publications

**Fouché S, Oggenfuss U, Chanclud E, Croll D**. 2022. A Devil's Bargain with transposable elements in plant pathogens. **Trends in Genetics** in press. [DOI](https://doi.org/10.1016/j.tig.2021.08.005)

**Tralamazza SM**, **Abraham L**, **Reyes Avila SC**, Reyes Correa B, **Croll D**. 2022. Histone H3K27 methylation perturbs transcriptional robustness and underpins dispensability of highly conserved genes in fungi. **Molecular Biology and Evolution** in press. [DOI](https://doi.org/10.1093/molbev/msab323)

**Oggenfuss U**, **Badet T**,  Wicker T, Hartmann FE, **Kumar Singh N**, **Abraham LN**, Karisto P, **Vonlanthen T**, Mundt CM, McDonald BA, **Croll D**. 2021. A population-level invasion by transposable elements triggers genome expansion in a fungal pathogen. **eLife** 10:e69249 [DOI](https://doi.org/10.7554/eLife.69249)

**Badet T, Fouché S, Hartmann FE**, Zala M, **Croll D**. 2021. Machine-learning predicts genomic determinants of meiosis-driven structural variation in a eukaryotic pathogen. **Nature Communications** 12:3551. [DOI](https://doi.org/10.1038/s41467-021-23862-x) (Open Access)

**Stauber L**, **Thomas B**, **Feurtey A**, Prospero S, **Croll D**. 2021. Emergence and diversification of a highly invasive chestnut pathogen lineage across south-eastern Europe. **eLife** 10:e56279. [DOI](https://doi.org/10.7554/eLife.56279) (Open Access)

**Dutta A**, **Hartmann FE**, Sardinha Francisco C, McDonald BA, **Croll D**. 2021. Mapping the adaptive landscape of a major agricultural pathogen reveals evolutionary constraints across heterogeneous environments. **The ISME Journal** 15:1402–1419. [DOI](https://doi.org/10.1038/s41396-020-00859-w) (Open Access)

Grossen C, Guillaume F, Keller LF, **Croll D**. 2020. Purging of highly deleterious mutations through severe bottlenecks in ibex. **Nature Communications** 11:1001. [DOI](https://doi.org/10.1038/s41467-020-14803-1) (Open Access)

**Fouche S**, **Badet T**, **Oggenfuss U**, **Plissonneau P**, Sardinha Francisco C, **Croll D**. 2020. Stress-driven transposable element de-repression dynamics and virulence evolution in a fungal pathogen. **Molecular Biology and Evolution** 37(1):221--239. [DOI](https://doi.org/10.1093/molbev/msz216). [Free Access](https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz216/5573762?guestAccessKey=d57f2891-c4a9-46f7-b0bc-d2272fa6b0cd).

Sánchez-Vallet A, **Fouché S**, Fudal I, **Hartmann FE**, Soyer JL, Tellier A, **Croll D**. 2018. The Genome Biology of Effector Gene Evolution in Filamentous Plant Pathogens. **Annual Review of Phytopathology** 56:21-40. [DOI](https://doi.org/10.1146/annurev-phyto-080516-035303). [Free PDF](http://www.annualreviews.org/eprint/ifv3kvkddKREwjuXz3Am/full/10.1146/annurev-phyto-080516-035303)

**Hartmann FE**, **Croll D**. 2018. Distinct Trajectories of Massive Recent Gene Gains and Losses in Populations of a Microbial Eukaryotic Pathogen. **Molecular Biology and Evolution** 34:2808-2822. [DOI](http://dx.doi.org/10.1093/molbev/msx208)

**Hartmann FE**, Sanchez-Vallet  A, McDonald BA, **Croll D**. 2017. A fungal wheat pathogen evolved host specialization by extensive chromosomal rearrangements. **The ISME Journal** 11:1189-1204. [DOI](http://dx.doi.org/10.1038/ismej.2016.196)


---  
## Preprints on BioRχiv
<i>(manuscripts already published in a journal are only listed further down)</i>

**Feurtey A**, Lorrain C, McDonald MC, Milgate A, Solomon P, Warren R, **Puccetti G**, Scalliet Gabriel, Torriani SFF, Gout L, Marcel TC, Suffert F, Alassimone J, Lipzen A, Yoshinaga Y, Daum C, Barry K, Grigoriev IV, Goodwin SB, Genissel A, Seidl MF, Stukenbrock H, Lebrun M, Kema GHJ, McDonald BA, **Croll D**. A thousand-genome panel retraces the global spread and climatic adaptation of a major crop pathogen. [DOI](https://doi.org/10.1101/2022.08.26.505378)

**Oggenfuss U**, **Croll D**. Recent transposable element bursts triggered by insertions near genes in a fungal pathogen. [DOI](https://doi.org/10.1101/2022.07.13.499862)

**Bellah H**, Gazeau G, Gelisse S, Amezrou R, Marcel TC, **Croll D**. A highly multiplexed assay to monitor virulence, fungicide resistance and gene flow in the fungal wheat pathogen _Zymoseptoria tritici_. [DOI](https://doi.org/10.1101/2022.07.18.500446)

**Monod V**, Hofstetter V, Zufferey V, Viret O, Gindro K, **Croll D**. Quantifying trade-offs in the choice of ribosomal barcoding markers for fungal amplicon sequencing: a case study on the grapevine trunk mycobiome. [DOI](https://doi.org/10.1101/2022.07.01.498490)

Latorre S\*, Were V\*, Foster AJ\*, Langner T\*, ..., **Reyes AS**, ..., **Croll D**, ..., Burbano HA, Kamoun S. A pandemic clonal lineage of the wheat blast fungus. [DOI](https://doi.org/10.1101/2022.06.06.494979) \*) joint first authors

**Kumar Singh N**\*, **Tralamazza S**\*, **Abraham LN**, Glauser G, **Croll D**. Genome-wide association mapping reveals genes underlying population-level metabolome diversity in a fungal crop pathogen. [DOI](https://doi.org/10.1101/2022.05.20.492824) \*) joint first authors

**Dutta A**, McDonald BA, **Croll D**. Combined reference-free and multi-reference approaches uncover cryptic variation underlying rapid adaptation in microbial pathogens. [DOI](https://doi.org/10.1101/2022.05.16.492091)

Castroagudin V, Danelli A, Intra Moreira S, Teodora de Assis Reges J, Carvalho G, Nunes Maciel JL, Variani Bonato AL, Forcelini CA, Alves E, McDonald BA, **Croll D**, Ceresini P. The wheat blast pathogen *Pyricularia graminis-tritici* has complex origins and a disease cycle spanning multiple grass hosts. [DOI](https://doi.org/10.1101/203455)

---    
## Preprints and datasets on Zenodo

**Croll D**. 2021. Whole-genome analyses of 286 *Magnaporthe oryzae* genomes suggest that an independent introduction of a global pandemic lineage is at the origin of the Zambia wheat blast outbreak. [DOI](https://doi.org/10.5281/zenodo.4655959)

Temba et al. 2021. Multiplex amplicon sequencing dataset for genotyping pandemic populations of the wheat blast fungus. [DOI](https://doi.org/10.5281/zenodo.4605959)

---    
## Publications 2022 and in press

116\. Çilingir FG, A’Bear L, Hansen D, Davis LR, Bunbury N, Ozgul A, **Croll D\***, Grossen C\*. Chromosome-level genome assembly for the Aldabra giant tortoise enables insights into the genetic health of a threatened population. **GigaScience** in press [bioRxiv version](https://doi.org/10.1101/2022.04.20.488802) \*) joint senior/corresponding authors

115\. Kretschmer M, Damoo D, Sun S, Lee C, Croll D, Brumer H, Kronstad J. Organic acids and glucose prime late stage fungal biotrophy in maize. **Science** 376(6598):1187-1191. [DOI](https://doi.org/10.1126/science.abo2401)

114\. Cornille A, Ebert D, Stukenbrock E, Rodriguez de la Vega R, Tiffin P, **Croll D\***, Tellier A\*. Unraveling coevolutionary dynamics using ecological genomics **Trends in Genetics** in press. [DOI](https://doi.org/10.1016/j.tig.2022.05.008) \*) joint senior/corresponding authors

113\. **Stauber L**, **Croll D**, Prospero S. Temporal changes in pathogen diversity in a perennial plant–pathogen–hyperparasite system. **Molecular Ecology** in press. [DOI](https://doi.org/10.1111/mec.16386)

112\. Cho YJ, Kim T, **Croll D**, Park M, Kim D, Keum HL, Sul WH, Jung WH. Genome of Malassezia arunalokei and its distribution on facial skin. **Microbiology Spectrum** in press. [DOI](https://doi.org/10.1128/spectrum.00506-22)

111\. **Stalder L**, **Oggenfuss U**, **Mohd-Assaad N**, **Croll D**. The population genetics of adaptation through copy-number variation in a fungal plant pathogen. **Molecular Ecology** in press. [DOI](https://doi.org/10.1111/mec.16435)

110\. John E, Jacques S, Phan HTT, Liu L, Pereira D, **Croll D**, Singh KB, Oliver RP, Tan KC. Variability in an effector gene promoter of a necrotrophic fungal pathogen dictates epistasis and effector-triggered susceptibility in wheat. **PLOS Pathogens** in press. [DOI](https://doi.org/10.1371/journal.ppat.1010149).

109\. **Fouché S, Oggenfuss U, Chanclud E, Croll D**. A Devil's Bargain with transposable elements in plant pathogens. **Trends in Genetics** in press. [DOI](https://doi.org/10.1016/j.tig.2021.08.005)

108\. Formenti G, Theissinger K, Fernandes C, ..., Miklós Bálint, The European Reference Genome Atlas (ERGA) Consortium (including **Croll D**). The era of reference genomes in conservation genomics. **Trends in Ecology & Evolution** in press. [DOI](https://doi.org/10.1016/j.tree.2021.11.008)

107\. **Tralamazza SM**, **Abraham L**, **Reyes Avila SC**, Reyes Correa B, **Croll D**. Histone H3K27 methylation perturbs transcriptional robustness and underpins dispensability of highly conserved genes in fungi. **Molecular Biology and Evolution** in press. [DOI](https://doi.org/10.1093/molbev/msab323)

106\. Reynard JS, Brodard J, Dubuis N, Kellenberger I, Spilmont AS, Roquis D, Maliogka V, Marchal C, Dedet S, **Gning O**, **Croll D**, Gindro K, Schumpp, O, Spring JL, Lacombe T. Screening of grapevine red blotch virus in two European ampelographic collections. **Journal of Plant Pathology** in press. [DOI](https://doi.org/10.1007/s42161-021-00952-9)

105\. **González-Sayer S**, **Oggenfuss U**, García I, Aristizabal F, **Croll D**\*, Riaño-Pachon DM\*. High-quality genome assembly of _Pseudocercospora ulei_ the main threat of natural rubber trees. **Genetics and Molecular Biology** in press. [DOI](https://doi.org/10.1590/1678-4685-GMB-2021-0051) \*joint senior authors

104\. **Kessler C**, Brambilla A, Waldvogel D, Camenisch G, Biebach I, Leigh DM, Grossen C, **Croll D**. A robust sequencing assay of a thousand amplicons for the high-throughput population monitoring of Alpine ibex immunogenetics. **Molecular Ecology Resources** in press. [DOI](https://doi.org/10.1111/1755-0998.13452)

103\. **Mossion V**, Dauphin B, **Grant J**, Zemp N, **Croll D**. A reference transcriptome for the early-branching fern *Botrychium lunaria* enables fine-grained resolution of population structure. **Molecular Ecology Resources** in press. [DOI](https://doi.org/10.1111/1755-0998.13478)

---    
## 2021

102\. **Oggenfuss U**, **Badet T**,  Wicker T, Hartmann FE, **Kumar Singh N**, **Abraham LN**, Karisto P, **Vonlanthen T**, Mundt CM, McDonald BA, **Croll D**. A population-level invasion by transposable elements triggers genome expansion in a fungal pathogen. **eLife** 10:e69249 [DOI](https://doi.org/10.7554/eLife.69249)

101\. Dauphin B, de Freitas Pereira M, Kohler A, Grigoriev IV, Barry K, Na H, Amirebrahimi M, Lipzen A, Martin F, Peter M, **Croll D**. Cryptic genetic structure and copy-number variation in the ubiquitous forest symbiotic fungus Cenococcum geophilum. **Environmental Microbiology** 23:6536-6556. [DOI](https://doi.org/10.1111/1462-2920.15752)

100\. **Singh NK**, Karisto P, **Croll D**. Population-level deep sequencing reveals the interplay of clonal and sexual reproduction in the fungal wheat pathogen *Zymoseptoria tritici*. **Microbial Genomics** 7:10. [DOI](https://doi.org/10.1099/mgen.0.000678)

99\. Cadot S, Gfeller V, Hu L, **Singh NK**, Sanchez-Vallet A, Glauser G, **Croll D**, Erb M, van der Heijden M, Schlaeppi K. Soil composition and plant genotype determine benzoxazinoid-mediated plant-soil feedbacks in cereals. **Plant, Cell & Environment** 44:3732-3744. [DOI](http://doi.org/10.1111/pce.14184)

98\. Crous P, Lombard L, Sandoval-Denis M, ..., **Croll D**, ..., Thines M. Fusarium: more than a node or a foot-shaped basal cell. **Studies in Mycology** 98:100116. [DOI](https://doi.org/10.1016/j.simyco.2021.100116)

97\. **Badet T, Fouché S, Hartmann FE**, Zala M, **Croll D**. Machine-learning predicts genomic determinants of meiosis-driven structural variation in a eukaryotic pathogen. **Nature Communications** 12:3551. [DOI](https://doi.org/10.1038/s41467-021-23862-x)

96\. **Singh NK**, **Badet T**, **Abraham LN**, **Croll D**. Rapid sequence evolution driven by transposable elements at a virulence locus in a fungal wheat pathogen. **BMC Genomics** 22:329. [DOI](https://doi.org/10.1186/s12864-021-07691-2)

95\. **Dutta A**, **Croll D**, McDonald BA, Krattinger SG. Genome-Wide Association Study Reveals Genetic Architecture of *Septoria Tritici* Blotch Resistance in a Historic Landrace Collection. **Euphytica** 217:108. [DOI](https://doi.org/10.1007/s10681-021-02843-7)

94\. **Stauber L**, **Thomas B**, Prospero S, **Croll D**. Emergence and diversification of a highly invasive chestnut pathogen lineage across south-eastern Europe. **eLife** 10:e56279. [DOI](https://doi.org/10.7554/eLife.56279)

93\. **Hendier A**, Chatelain C, Du Pasquier PE, Paris M, Ouattara K, Koné I, **Croll D**, Zuberbühler K. A new method to determine the diet of pygmy hippopotamus in Taï National Park, Côte d’Ivoire. **African Journal of Ecology** 59:809-825 [DOI](https://doi.org/10.1111/aje.12888)

92\. **Croll D**, Crous PW, Pereira D, Mordecai EA, McDonald BA, Brunner PC. Genome-scale phylogenies reveal relationships among *Parastagonospora* species infecting domesticated and wild grasses. **Persoonia** 46:116-128 [DOI](https://doi.org/10.3767/persoonia.2021.46.04)

91\. Pereira D, **Oggenfuss U**, McDonald BA, **Croll D**. The population genomics of transposable element activation in the highly repressive genome of an agricultural pathogen. **Microbial Genomics** 7:8. [DOI](https://doi.org/10.1099/mgen.0.000540)

90\. Lorrain C, **Oggenfuss U**, **Croll D**, Duplessis S, Stukenbrock E. Transposable Elements in Fungi: Coevolution With the Host Genome Shapes Genome Architecture, Plasticity and Adaptation. Chapter in **Encyclopedia of Mycology**. Elsevier. ISBN 9780128199909. [Link](https://www.elsevier.com/books/encyclopedia-of-mycology/zaragoza/978-0-12-819990-9)

89\. **Singh NK**, **Dutta A**, **Puccetti G**, **Croll D**. Tackling microbial threats in agriculture with integrative imaging and computational approaches. **Computational and Structural Biotechnology Journal** 19:372-383. [DOI](https://doi.org/10.1016/j.csbj.2020.12.018) (Open Access)

88\. **Dutta A**, **Hartmann FE**, Sardinha Francisco C, McDonald BA, **Croll D**. Mapping the adaptive landscape of a major agricultural pathogen reveals evolutionary constraints across heterogeneous environments. **The ISME Journal** 15:1402–1419. [DOI](https://doi.org/10.1038/s41396-020-00859-w) (Open Access)

87\. **Hartmann FE**, **Vonlanthen T**, **Singh NK**, McDonald M, Milgate A, **Croll D**. The complex genomic basis of rapid convergent adaptation to pesticides across continents in a fungal plant pathogen. **Molecular Ecology**. [DOI](https://doi.org/10.1111/mec.15737) (Open Access)

86\. **Dutta A**, **Croll D**, McDonald BA, Barrett LG. Maintenance of variation in virulence and reproduction in populations of an agricultural plant pathogen. **Evolutionary Applications**. [DOI](https://doi.org/10.1111/eva.13117) (Open Access)


---    
## 2020

85\. Pereira D, McDonald BA, **Croll D**. The genetic architecture of emerging fungicide resistance in populations of a global wheat pathogen. **Genome Biology and Evolution** 12(12):2231–2244. [DOI](https://doi.org/10.1093/gbe/evaa203) (Open Access)

84\. **Stauber L**, Prospero S, **Croll D**. Comparative genomics analyses of lifestyle transitions at the origin of an invasive fungal pathogen in the genus *Cryphonectria*. **mSphere** 5:e00737-20. [DOI](https://doi.org/10.1128/mSphere.00737-20) (Open Access)

83\. Fokkens L+, Guo L+, Dora S, Wang B, Ye K\*, Sanchez-Rodriguez C\*, **Croll D**\*. A chromosome-scale genome assembly for the *Fusarium oxysporum* strain Fo5176 to establish a model Arabidopsis-fungal pathosystem. **G3: Genes\|Genomes\|Genetics** 10(10):3549-3555. [DOI](https://doi.org/10.1534/g3.120.401375) (Open Access) +) joint first, \*) joint senior/corresponding authors

82\. Torres DE+, **Oggenfuss U+**, **Croll D**\*, Seidl MF\*. Genome evolution in fungal plant pathogens: looking beyond the two-speed genome model. **Fungal Biology Reviews** 34(3):136-143. [DOI](https://doi.org/10.1016/j.fbr.2020.07.001) (Open Access). +) joint first, \*) joint senior/corresponding authors

81\. Vesga P, Flury P, Vacheron J, Keel C\*, **Croll D**\*, Maurhofer M\*. Transcriptome plasticity underlying plant root colonization and insect invasion by *Pseudomonas protegens*. **The ISME Journal** 14:2766–2782. [DOI](https://doi.org/10.1038/s41396-020-0729-9) (Open Access) \*) joint senior/corresponding authors

80\. Feurtey A, Lorrain C, **Croll D**, Eschenbrenner C, Freitag M, Habig M, Haueisen J, Moller M, Schotanus K, Stukenbrock EH. Genome compartmentalization predates species divergence in the plant pathogen genus *Zymoseptoria*. **BMC Genomics**  21(588). [DOI](https://doi.org/10.1186/s12864-020-06871-w) (Open Access)

79\. Sánchez-Vallet A, Tian H, Rodriguez-Moreno L, Valkenburg DJ, Saleem-Batcha R, Wawra S, Kombrink A, Verhage L, de Jonge R, van Esse HP,  Zuccaro A,  **Croll D**, Mesters JR, Thomma BPHJ. A secreted LysM effector protects fungal hyphae through chitin-dependent homodimer polymerization. **PLOS Pathogens** 16(6):e1008652. [DOI](https://doi.org/10.1371/journal.ppat.1008652) (Open Access)

78\. **Badet T**, **Croll D**. The rise and fall of genes: origins and functions of plant pathogen pangenomes. **Current Opinion in Plant Biology** 56:65-73. [DOI](https://doi.org/10.1016/j.pbi.2020.04.009) (Open Access)

77\. Aubry S, **Reyes Avila SC**, **Croll D**, Christ B. Omics-based detection, identification and quantification of GM food and feed: current challenges and perspectives. Chapter in **Foodomics**. Royal Society of Chemistry. ISBN 978-1-78801-884-5. [Link](https://pubs.rsc.org/en/content/ebook/978-1-78801-884-5)

76\. Pereira D, **Croll D**, Brunner P, McDonald BA. Natural selection drives population divergence for local adaptation in a wheat pathogen. **Fungal Genetics & Biology** 141:103398. [DOI](https://doi.org/10.1016/j.fgb.2020.103398)

75\. Heinzelmann R, Rigling R, Sipos G, Münsterkötter M, **Croll D**. Chromosomal assembly and analyses of genome-wide recombination rates in the forest pathogenic fungus *Armillaria ostoyae*. **Heredity** 124:699–713. [DOI](https://doi.org/10.1038/s41437-020-0306-z)

74\. Grossen C, Guillaume F, Keller LF, **Croll D**. Purging of highly deleterious mutations through severe bottlenecks in ibex. **Nature Communications** 11:1001. [DOI](https://doi.org/10.1038/s41467-020-14803-1) (Open Access)

73\. **Badet T**, **Oggenfuss U**, **Abraham LN**, McDonald BA, **Croll D**. A 19-isolate reference-quality global pangenome for the fungal wheat pathogen *Zymoseptoria tritici*. **BMC Biology** 18:12. [DOI](https://doi.org/10.1186/s12915-020-0744-3) (Open Access)

72\. **Fouche S**, **Badet T**, **Oggenfuss U**, **Plissonneau P**, Sardinha Francisco C, **Croll D**. Stress-driven transposable element de-repression dynamics and virulence evolution in a fungal pathogen. **Molecular Biology and Evolution** 37(1):221--239. [DOI](https://doi.org/10.1093/molbev/msz216). [Free Access](https://academic.oup.com/mbe/advance-article/doi/10.1093/molbev/msz216/5573762?guestAccessKey=d57f2891-c4a9-46f7-b0bc-d2272fa6b0cd).


---    
## 2019

71\. **Tralamazza S**, Oliveira Rocha L, **Oggenfuss U**, Correa B, **Croll D**. Complex evolutionary origins of specialized metabolite gene cluster diversity among the plant pathogenic fungi of the *Fusarium graminearum *species complex. **Genome Biology and Evolution** 11:3106--3122. [DOI](https://doi.org/10.1093/gbe/evz225) (Open Access)

70\. **Mohd-Assaad N**, McDonald BA, **Croll D**. The emergence of the multi-species NIP1 effector in *Rhynchosporium* was accompanied by high rates of gene duplications and losses. **Environmental Microbiology** 21: 2677-2695. [DOI](https://doi.org/10.1111/1462-2920.14583)

69\. McDonald M, Renkin M, Spackman M, Orchard B, **Croll D**, Solomon P, Milgate A. Rapid parallel evolution of azole fungicide resistance in Australian populations of the wheat pathogen *Zymoseptoria tritici*. **Applied and Environmental Microbiology** 85(4):e01908-18. [DOI](https://doi.org/10.1128/AEM.01908-18)

68\. Gupta DR, Sarai CRA, Win J, Soanes DM, Ryder LS, **Croll D**, Bhattacharjee P, Hossain MS, Mahmud NU, Mehbub MS, Surovy MZ, Talbot NJ, Kamoun S, Islam I. Cautionary Notes on Use of the MoT3 Diagnostic Assay for *Magnaporthe oryzae* Wheat and Rice Blast Isolates. **Phytopathology** 109(4):504--508. [DOI](https://doi.org/10.1094/PHYTO-06-18-0199-LE)

67\. Ceresini PC, Castroagudín VL, Rodrigues FA, Rios JA, Aucique-Pérez CE, Moreira SI, **Croll D**, Alves E, de Carvalho G, Nunes Maciel JL, McDonald BA. Wheat blast: from its origins in South America to its emergence as a global threat. **Molecular Plant Pathology** 20:155-172. [DOI](https://doi.org/10.1111/mpp.12747)


---    
## 2018

66\. Attarian R, Hu G, Sánchez-León E, Caza M, **Croll D**, Do E, Bach H, Missall T, Lodge J, Jung WH, Kronstad J. The monothiol glutaredoxin Grx4 regulates iron homeostasis and virulence in *Cryptococcus neoformans*. **mBio** 9(6):e02377-18. [DOI](https://doi.org/10.1128/mBio.02377-18) (Open Access)

65\. Krishnan P, Meile L, **Plissonneau C**, Ma X, **Hartmann FE**, **Croll D**, McDonald BA, Sánchez-Vallet A. Transposable element insertions shape gene regulation and melanin production in a fungal pathogen of wheat. **BMC Biology** 16:78. [DOI](https://doi.org/10.1186/s12915-018-0543-2) (Open Access)

64\. **Fouche S**, **Plissonneau C**, McDonald BA, **Croll D**. Meiosis leads to pervasive copy-number variation and distorted inheritance of accessory chromosomes of the wheat pathogen *Zymoseptoria tritici*. **Genome Biology and Evolution** 10:1416--1429. [DOI](https://doi.org/10.1093/gbe/evy100) (Open Access)

63\. **Mohd-Assaad N**, McDonald BA, **Croll D**. Genome-wide detection of genes under positive selection in worldwide populations of the barley scald pathogen. **Genome Biology and Evolution** 10:1315--1332. [DOI](https://doi.org/10.1093/gbe/evy087) (Open Access)

62\. **Hartmann FE**, McDonald BA, **Croll D**. Genome-wide evidence for divergent selection between populations of a major agricultural pathogen. **Molecular Ecology**. 27:2725-2741. [DOI](https://doi.org/10.1111/mec.14711)

61\. Sánchez-Vallet A, **Fouché S**, Fudal I, **Hartmann FE**, Soyer JL, Tellier A, **Croll D**. The Genome Biology of Effector Gene Evolution in Filamentous Plant Pathogens. **Annual Review of Phytopathology** 56:21-40. [DOI](https://doi.org/10.1146/annurev-phyto-080516-035303). [Free PDF](http://www.annualreviews.org/eprint/ifv3kvkddKREwjuXz3Am/full/10.1146/annurev-phyto-080516-035303)

60\. Ceresini PC, Castroagudín VL, Rodrigues FA, Rios JA, Aucique-Pérez CE, Moreira SI, Alves E, **Croll D**, Nunes Maciel JL. Wheat Blast: Past, Present, and Future. **Annual Review of Phytopathology** 56:427-456. [DOI](https://doi.org/10.1146/annurev-phyto-080417-050036). [Free PDF](http://www.annualreviews.org/eprint/FDrqhEx9su3gfR5YKKzS/full/10.1146/annurev-phyto-080417-050036)

59\. **Benevenuto J**, Texeira-Silva NS, Kuramae EE, **Croll D**\*, Vitorello CBM\*. Comparative genomics of smut pathogens: insights from orphans and positively selected genes into host specialization. **Frontiers in Microbiology** 06 April. [DOI](https://doi.org/10.3389/fmicb.2018.00660) (Open Access) \*) joint senior/corresponding authors

58\. Meile L, **Croll D**, Brunner PC, **Plissonneau C**, **Hartmann FE**, McDonald BA, Sanchez-Vallet* A. A fungal avirulence factor encoded in a highly plastic genomic region triggers partial resistance to septoria tritici blotch. **New Phytologist** 219:1048--1061. [DOI](https://doi.org/10.1111/nph.15180)

57\. Schirrmann MK, Zoller S, **Croll D**, Stukenbrock EH, Leuchtmann A, Fior S.  Genome-wide signatures of selection in *Epichloë* reveal candidate genes for host specialization. **Molecular Ecology** 27(15):3070-3086 [DOI](https://doi.org/10.1111/mec.14585)

56\. **Fouché S**, **Plissonneau C**, **Croll D**. The birth and death of effectors in rapidly evolving filamentous pathogen genomes. **Current Opinion in Microbiology** 46:34-42. [DOI](https://doi.org/10.1016/j.mib.2018.01.020) [PDF](https://authors.elsevier.com/a/1Wc7a4tPFpD~QO)

55\. **Plissonneau C**, **Hartmann FE**, **Croll D**. Pangenome analyses of the wheat pathogen *Zymoseptoria tritici* reveal the structural basis of a highly plastic eukaryotic genome. **BMC Biology** 16:5. [DOI](https://doi.org/10.1186/s12915-017-0457-4) (Open Access)

54\. Kretschmer M, Lambie S, **Croll D**, Kronstad JW. Acetate provokes mitochondrial stress and cell death in *Ustilago maydis*. **Molecular Microbiology** . 107:488-507. [DOI](http://dx.doi.org/10.1111/mmi.13894)

53\. Sanchez-Vallet  A, **Hartmann FE**, Marcel T, **Croll D**. Nature's genetic screens: using genome-wide association studies for effector discovery. **Molecular Plant Pathology** . 19:3-6. [DOI](http://dx.doi.org/10.1111/mpp.12592)

52\. Grossen C, Biebach I, Angelone-Alasaad S, Keller LF, **Croll D**. Population genomics analyses of European ibex species show lower diversity and higher inbreeding in reintroduced populations. **Evolutionary Applications** . 11:123--139. [DOI](http://dx.doi.org/10.1111/eva.12490)

51\. Stewart EL, **Croll D**, Lendenmann MH, Sanchez Vallet A, **Hartmann FE**, Palma Guerrero J, Ma X, McDonald BA. QTL mapping reveals complex genetic architecture of quantitative virulence in the wheat pathogen *Zymoseptoria tritici*. **Molecular Plant Pathology** 19:201-216. [DOI](http://dx.doi.org/10.1111/mpp.12515)


---    
## 2017

50\. Heinzelmann R, **Croll D**, Zoller S, Sipos G, Münsterkötter M, Güldener U, Rigling D. High-density genetic mapping identifies the genetic basis of a natural colony morphology mutant in the root rot pathogen *Armillaria ostoyae.* **Fungal Genetics & Biology** 108:44-54. [DOI](http://dx.doi.org/10.1016/j.fgb.2017.08.007)

49\. **Hartmann FE**, **Croll D**. Distinct Trajectories of Massive Recent Gene Gains and Losses in Populations of a Microbial Eukaryotic Pathogen. **Molecular Biology and Evolution** 34:2808-2822. [DOI](http://dx.doi.org/10.1093/molbev/msx208)

48\. **Plissonneau C**, **Benevenuto J**, **Mohd-Assaad N**, **Fouché S**, **Hartmann FE**, **Croll D**. Using population and comparative genomics to understand the genetic basis of effector-driven fungal pathogen evolution. **Frontiers in Plant Science** 8:119. [DOI](http://journal.frontiersin.org/article/10.3389/fpls.2017.00119/full) [PDF](http://www.pathogen-genomics.org/uploads/1/9/9/1/19914417/fpls-08-00119.pdf)

47\. Palma-Guerrero J, Ma X, Torriani SFF, Zala M, Francisco CS, **Hartmann FE**, **Croll D**, McDonald BA. Comparative transcriptome analyses in *Zymoseptoria tritici* reveal significant differences in gene expression among strains during plant infection. **Molecular Plant-Microbe Interactions**. 30:231-244. [DOI](http://dx.doi.org/10.1094/MPMI-07-16-0146-R)

46\. Zhong Z+, Marcel T+, **Hartmann FE**, Ma X, **Plissonneau C**, Zala M, Ducasse A, Confais J, Compain J, Lapalu N, Amselem J, McDonald BA, **Croll D**\*, Palma-Guerrero J\*. A small secreted protein in *Zymoseptoria tritici* is responsible for avirulence on wheat cultivars carrying the *Stb6* resistance gene. **New Phytologist** 214:619-631  +) joint first authors \*) joint senior/corresponding authors. [DOI](http://dx.doi.org/10.1111/nph.14434)

45\. Kokko H, Chaturvedi A, **Croll D**, Fischer MC, Guillaume F, Karrenberg S, Kerr B, Rolshausen G, Stapley J. Can evolution supply what ecology demands? **Trends in Ecology & Evolution**. 32:187-197. [DOI](http://dx.doi.org/10.1016/j.tree.2016.12.005)

44\. **Hartmann FE**, Sanchez-Vallet  A, McDonald BA, **Croll D**. A fungal wheat pathogen evolved host specialization by extensive chromosomal rearrangements. **The ISME Journal**. 11:1189-1204. [DOI](http://dx.doi.org/10.1038/ismej.2016.196)

43\. **Croll D**, McDonald BA. The genetic basis of local adaptation for pathogenic fungi in agricultural ecosystems. **Molecular Ecology** . 26:2027-2040. [DOI](http://dx.doi.org/10.1111/mec.13870)

42\. Kretschmer M, **Croll D**, Kronstad JW. Chloroplast-associated metabolic functions influence the susceptibility of maize to *Ustilago maydis*. **Molecular Plant Pathology**. 18:1210-1221. [DOI](http://dx.doi.org/%2010.1111/mpp.12485)

41\. Kretschmer M, **Croll D**, Kronstad JW. Maize susceptibility to *Ustilago maydis* is influenced by genetic and chemical perturbation of carbohydrate allocation. **Molecular Plant Pathology**. 18:1222-1237. [DOI](http://dx.doi.org/%2010.1111/mpp.12486)

40\. Lambie S, Kretschmer M, **Croll D**, Haslam TM, Kunst L, Klose J, Kronstad JW. The putative phospholipase Lip2 counteracts oxidative damage and influences the virulence of *Ustilago maydis*. **Molecular Plant Pathology** 18(2):210--221. [DOI](http://dx.doi.org/10.1111/mpp.12391)


---    
## 2016

39\. **Mohd-Assaad N**, McDonald BA, **Croll D**. Multi-locus resistance evolution to azole fungicides in fungal plant pathogen populations. **Molecular Ecology** 25:6124--6142 . [DOI](http://dx.doi.org/10.1111/mec.13916)

38\. **Plissonneau C**, **Stürchler A**, **Croll D**. The evolution of orphan regions in genomes of a fungal pathogen of wheat. **mBio** 7 (5):e01231-16. [DOI](http://dx.doi.org/10.1128/mBio.01231-16) [PDF](http://www.pathogen-genomics.org/uploads/1/9/9/1/19914417/mbio-2016-plissonneau-.pdf)

37\. **Croll D**. Evolution in the absence of sex: Ideas revisited in the post-genomics age. **BioEssays** 38 (12): 1191. [DOI](http://dx.doi.org/10.1002/bies.201600195)

36\. Islam TM, **Croll D**, Gladieux P, Soanes DM, Persoons A, Bhattacharjee P, Hossain S, Gupta DR, Rahman M, Mahboob GM, Cook N, Salam MU, Surovy MZ, Sancho VB, Maciel JLN, Júnior AN, Castroagudín VL, de Assis Reges JT, Ceresini PC, Ravel S, Kellner R, Fournier E, Tharreau D, Lebrun MH, McDonald BA, Stitt T, Swan D, Talbot NJ, Saunders DGO, Win J, Kamoun S. Emergence of wheat blast in Bangladesh was caused by a South American lineage of *Magnaporthe oryzae*. **BMC Biology** 14:84. [DOI](http://dx.doi.org/10.1186/s12915-016-0309-7)

35\. **Croll D**, Laine AL. What the population genetic structures of host and pathogen tell us about disease evolution. **New Phytologist**. 212 (3): 537--539. [DOI](http://dx.doi.org/10.1111/nph.14203)

34\. Grossen C, Seneviratne S, **Croll D**, Irwin DE. Strong reproductive isolation and narrow genomic tracts of differentiation among three woodpecker species in secondary contact. **Molecular Ecology**. 25 (17): 4247--4266. [DOI](http://dx.doi.org/%2010.1111/mec.13751)

33\. Palma-Guerrero J, Torriani SFF, Zala M, Carter D, Courbot M, Rudd JJ, McDonald BA, **Croll D**. Comparative transcriptome analyses of *Zymoseptoria tritici* strains show complex lifestyle transitions and intra-specific variability in transcription profiles. **Molecular Plant Pathology**.  17 (6): 845--859. [DOI](http://dx.doi.org/10.1111/mpp.12333)

32\. Lendenmann M, **Croll D**, Palma-Guerrero J, Stewart E, McDonald BA. QTL Mapping of Temperature Sensitivity Reveals Candidate Genes for Thermal Adaptation and Growth Morphology in the Plant Pathogenic Fungus *Zymoseptoria tritici*. **Heredity** 116:384-394. [DOI](http://dx.doi.org/10.1038/hdy.2015.111)

31\. Geddes JMH, Caza M, **Croll D**, Stoynov N, Foster LJ, Kronstad JW. Analysis of the Protein Kinase A-regulated proteome of *Cryptococcus neoformans* identifies a role for the ubiquitin-proteasome pathway in capsule formation. **mBio** 7 (1): e01862-15. [DOI](http://dx.doi.org/10.1128/mBio.01862-15) [PDF](http://www.pathogen-genomics.org/uploads/1/9/9/1/19914417/mbio_2016_geddes.pdf)


---    
## 2015

30\. **Croll D**, Lendenmann MH, Stewart E, McDonald BA. Impact of recombination hotspots on genome evolution of a fungal plant pathogen. **Genetics** 201(3):1213-1228. [DOI](http://dx.doi.org/10.1534/genetics.115.180968)

29\. Geddes JMH, **Croll D**, Caza M, Stoynov N, Foster LJ, Kronstad JW. Secretome Profiling of *Cryptococcus neoformans* Reveals Regulation of a Subset of Virulence-Associated Proteins and Potential Biomarkers by Protein Kinase A. **BMC Microbiology** 15 (1):206. [DOI](http://dx.doi.org/10.1186/s12866-015-0532-3) [PDF](http://www.pathogen-genomics.org/uploads/1/9/9/1/19914417/bmc_microbiol._2015_geddes.pdf)

28\. Lendenmann MH, **Croll D**, McDonald BA. QTL Mapping of Fungicide Sensitivity Reveals Novel Genes and Pleiotropy with Melanization in the Pathogen *Zymoseptoria tritici.* **Fungal Genetics & Biology** 80:53-67. [DOI](http://dx.doi.org/10.1016/j.fgb.2015.05.001)

27\. Jung WH, **Croll D**, Cho JH, Kim YR, Lee YW. Analysis of the Nasal Vestibule Mycobiome in Patients with Allergic Rhinitis. **Mycoses** 58 (3):167-172. [DOI](http://dx.doi.org/10.1111/myc.12296)


---    
## 2014

26\. Lendenmann MH, **Croll D**, Stewart EL, McDonald BA. Quantitative Trait Locus Mapping of Melanization in the Plant Pathogenic Fungus *Zymoseptoria tritici*. **G3:Genes\|Genomes\|Genetics** 4(12):2519-2533.[DOI](http://dx.doi.org/10.1534/g3.114.015289) [PDF](http://www.pathogen-genomics.org/uploads/1/9/9/1/19914417/g3__bethesda__2014_lendenmann.pdf)

25\. Billmyre B, **Croll D**, Li W, Mieczkowski P, Carter DA, Cuomo C, Kronstad JW, Heitman J. Highly recombinant VGII *Cryptococcus gattii* population develops clonal outbreak clusters through both sexual macroevolution and asexual microevolution. **mBio **5 (4): e01494-14.[DOI](http://dx.doi.org/10.1128/mBio.01494-14) [PDF](http://www.pathogen-genomics.org/uploads/1/9/9/1/19914417/mbio_2014_billmyre.pdf)

24\. Grossen C, Keller L, Biebach I, The International Goat Genome Consortium, **Croll D**. Introgression from domestic goat generated variation at the major histocompatibility complex of Alpine ibex. **PLoS Genetics** 10(6):e1004438 [DOI](http://www.plosgenetics.org/doi/pgen.1004438) [PDF](http://www.pathogen-genomics.org/uploads/1/9/9/1/19914417/plos_genet_2014_grossen.pdf)

23\. Brunner PC, Torriani SFF, **Croll D**, Stukenbrock EH, McDonald BA. Hitchhiking selection is driving intron gain in a pathogenic fungus. **Molecular Biology and Evolution** 31(7):1741-1749[DOI](http://dx.doi.org/10.1093/molbev/msu123)

22\. Stukenbrock E, **Croll D**. The evolving fungal genome. **Fungal Biology Reviews** 28:1-12[DOI](http://dx.doi.org/10.1016/j.fbr.2014.02.001)

21\. Stefansson T, Willi  Y, **Croll D**, McDonald BA. An assay for quantitative virulence in *Rhynchosporium commune* reveals an association between effector genotype and virulence. **Plant Pathology** 63:405-414.  [DOI](http://dx.doi.org/10.1111/ppa.12111)

20\. **Croll D**. Learn from the fungi: Adaptive evolution without sex in fungal pathogens. **BioEssays** 36:334  [DOI](http://dx.doi.org/10.1002/bies.201400030)


---    
## 2013

19\. **Croll D**, Zala M, McDonald BA. Breakage-fusion-bridge cycles and large insertions contribute to the rapid evolution of accessory chromosomes in a fungal pathogen. **PLoS Genetics** 9 (6):e1003567. [DOI](http://dx.doi.org/10.1371/journal.pgen.1003567) [PDF](http://www.pathogen-genomics.org/uploads/1/9/9/1/19914417/plos_genet_2013_croll-small.pdf)

18\. Brunner PC, Torriani SFF, **Croll D**, Stukenbrock EH, McDonald BA. Coevolution and life cycle specialization of plant cell wall degrading enzymes in a hemibiotrophic pathogen. **Molecular Biology and Evolution** 30:1337--1347. [DOI](http://dx.doi.org/10.1093/molbev/mst041)

17\. Drabešová J, Ryšánek P, Brunner PC, McDonald BA, **Croll D**. 2013. Population genetic structure of *Mycosphaerella graminicola* and Quinone Outside Inhibitor (QoI) resistance in the Czech Republic.** European Journal of Plant Pathology **135:211--224. [DOI](http://dx.doi.org/10.1007/s10658-012-0080-8)


---    
## 2012

16\. **Croll D**, McDonald BA. 2012. Intron gains and losses in the evolution of *Fusarium* and *Cryptococcus* fungi. **Genome Biology and Evolution** 4:1148-1161. [DOI](http://dx.doi.org/10.1093/gbe/evs091) [PDF](http://www.pathogen-genomics.org/uploads/1/9/9/1/19914417/genome_biol_evol_2012_croll.pdf)

15\. Ehinger M, **Croll D**, Koch AM, Sanders IR. 2012 . Significant genetic and phenotypic changes arising from clonal growth of a single spore of an arbuscular mycorrhizal fungus over multiple generations. **New Phytologist** 196:853-861. [DOI](http://dx.doi.org/10.1111/j.1469-8137.2011.03948.x)

14\. **Croll D**, McDonald BA. 2012 The accessory genome as a cradle for adaptive evolution in pathogens.** PLoS Pathogens** 8(4):e1002608. [DOI](http://dx.plos.org/10.1371/journal.ppat.1002608) [PDF](http://www.pathogen-genomics.org/uploads/1/9/9/1/19914417/plos_pathog_2012_croll.pdf)

13\. Hofstetter V, Buyck B, **Croll D**, Viret O, Couloux A, Gindro K. 2012. What if esca disease of grapevine were not a fungal disease? **Fungal Diversity** 54:51-67. [DOI](http://dx.doi.org/10.1007/s13225-012-0171-z)

12\. Tisserant E, Kohler A, Dozolme-Seddas P, Balestrini R, Benabdellah K, Colard A, **Croll D**, Da Silva C, Gomez SK, Koul R, et al. 2012. The transcriptome of the arbuscular mycorrhizal fungus *Glomus intraradices* (DAOM 197198) reveals functional tradeoffs in an obligate symbiont. **New Phytologist** 193:755-769. [DOI](http://dx.doi.org/10.1111/j.1469-8137.2011.03948.x)


---    
## 2011

11\. Torriani SFF, Stukenbrock EH, Brunner PC, McDonald BA, **Croll D**. 2011. Evidence for Extensive Recent Intron Transposition in Closely Related Fungi. **Current Biology** 21:2017-2022. [DOI](http://dx.doi.org/10.1016/j.cub.2011.10.041)

10\. Stöck M, **Croll D**, Dumas Z, Biollay S, Wang J, Perrin N. 2011. A cryptic heterogametic transition revealed by sex-linked DNA markers in Palearctic green toads.** Journal Of Evolutionary Biology** 24:1064-1070. [DOI](http://dx.doi.org/10.1111/j.1420-9101.2011.02239.x)


---    
## 2010

9\. Sanders IR, **Croll D**. 2010. Arbuscular mycorrhiza: the challenge to understand the genetics of the fungal partner. **Annual Review of Genetics** 44:271-292. [DOI](http://dx.doi.org/10.1146/annurev-genet-102108-134239)

8\. Angelard C, Colard A, Niculita-Hirzel H, **Croll D**, Sanders IR. 2010. Segregation in a mycorrhizal fungus alters rice growth and symbiosis-specific gene transcription. **Current Biology** 20:1216-1221. [DOI](http://dx.doi.org/10.1016/j.cub.2010.05.031)


---    
## 2009

7\. **Croll D**, Sanders IR. 2009. Recombination in *Glomus intraradices*, a supposed ancient asexual arbuscular mycorrhizal fungus. **BMC Evolutionary Biology** 9:13. [DOI](http://dx.doi.org/10.1186/1471-2148-9-13)

6\. Corradi N, Ruffner B, **Croll D**, Colard A, Horák A, Sanders IR. 2009. High-level molecular diversity of copper-zinc superoxide dismutase genes among and within species of arbuscular mycorrhizal fungi. **Applied and Environmental Microbiology** 75:1970-1978. [DOI](http://dx.doi.org/10.1128/AEM.01974-08)

5\. **Croll D**, Giovannetti M, Koch AM, Sbrana C, Ehinger M, Lammers PJ, Sanders IR. 2009. Nonself vegetative fusion and genetic exchange in the arbuscular mycorrhizal fungus *Glomus intraradices.* **New Phytologist** 181:924-937. [DOI](http://dx.doi.org/10.1111/j.1469-8137.2008.02726.x)


---    
## - 2008

4\. **Croll D**, Corradi N, Gamper HA, Sanders IR. 2008. Multilocus genotyping of arbuscular mycorrhizal fungi and marker suitability for population genetics. **New Phytologist** 180:564-568. [DOI](http://dx.doi.org/10.1111/j.1469-8137.2008.02602.x)

3\. **Croll D**, Wille L, Gamper HA, Mathimaran N, Lammers PJ, Corradi N, Sanders IR. 2008. Genetic diversity and host plant preferences revealed by simple sequence repeat and mitochondrial markers in a population of the arbuscular mycorrhizal fungus *Glomus intraradices*. **New Phytologist** 178:672-687. [DOI](http://dx.doi.org/10.1111/j.1469-8137.2008.02381.x)

2\. Corradi N, **Croll D**, Colard A, Kuhn G, Ehinger M, Sanders IR. 2007. Gene copy number polymorphisms in an arbuscular mycorrhizal fungal population. **Applied and Environmental Microbiology** 73:366-369. [DOI](http://dx.doi.org/10.1128/AEM.01574-06)

1\. Koch AM, **Croll D**, Sanders IR. 2006. Genetic variability in a population of arbuscular mycorrhizal fungi causes variation in plant growth. **Ecology Letters** 9:103-110. [DOI](http://dx.doi.org/10.1111/j.1461-0248.2005.00853.x)
