---
layout: post
title:  "Untangling complex relationships"
date:   2019-08-05
categories: lab life
---

*Blog post written by Luzia Stalder who started her PhD in our lab in spring 2019.*

Understanding how characteristics of organisms arise and interact with each other – this is the fundamental aim of biological research. As researchers, we want to unveil the complex relationships between physiological, molecular and genetic traits. Importantly, these relationships evolved in a highly branched network. Thus, to untangle complex relationships, we need to be able to measure a large set of variables at different levels simultaneously. For example, to understand the virulence of a pathogen on a host, we require knowledge of genomes of pathogen strains with different virulence, and test how these affect different hosts, which themselves are in different environments and inhabit different microbes. I believe that large datasets with information collections at different levels – together with modern bioinformatic tools – are key to uncover complex relationships of biological networks in a systematic manner.

Since I started to work on biological questions, I was fascinated how new meaningful relationships are revealed using large datasets. During my master studies at ETH Zurich, I extracted complex relationships from high-throughput genomic, transcriptomic - and proteomic data. In my first project, I used genomic data collected around the world with 114 populations of Rhynchosporium commune, a major fungal pathogen of barley, to understand how genomic changes influence its adaptation to fungicide. For my second project, I used transcriptomic datasets of hippocampal brain tissue of mice that have been subjected to either a stressful or a non-stressful situation, to understand how stress influences the expression network of different hippocampal regions. In a third project, I used large proteomic datasets of yeast, fly, mouse and human to understand how basic protein networks evolve across evolution.

For my PhD project, I joined the lab of Daniel Croll at the University of Neuchâtel, to focus on the complex microbiome network that influences plant health and disease. Typically, researchers study pathogen- host interactions in an isolated one pathogen – one host system. However, it has been shown that the presence of a second microbe on the host plant can either aid the arriving pathogen in a way that it becomes far more virulent – or on the contrary, can prevent the arriving pathogen from infecting the plant completely. One prominent example of such microbes are Pseudomonas bacteria, which are omnipresent on plant leaves and roots. Different Pseudomonas species either promote or inhibit fungal pathogen growth, and therefore significantly affect plant disease. I am intrigued by such microbe-microbe interactions and their impact on plant health. I hope to untangle a part of this complex interplay in the coming years. For this, I will focus on the tripartite system of wheat, the fungus Zymoseptoria tritici - the major pathogen of wheat - and the bacteria Pseudomonas. I want to understand how the diverse bacteria-fungal interactions affect plant health on a genetic and functional level. I will use high-throughput genomic and transcriptomic approaches to do so. My ultimate goal is to understand complex relationships that can be used for sustainable crop production in the first place, but also allow to draw parallels to complex microbiome systems in human.



[Back to all blogs](/blog/)
